utils/metrics.py

```diff
@@ -1,7 +1,8 @@
-from sklearn.metrics import classification_report
 import numpy as np
+from sklearn.metrics import classification_report, matthews_corrcoef, recall_score
+from math import sqrt
```

Comment:
Instead of using the math package, use the `np.sqrt`, it will avoid unnecessary import.